modules/configuration/src/test/java/org/apache/ignite/internal/configuration/ConfigurationChangerTest.java

```diff
@@ -95,12 +96,12 @@ public static class FirstConfigurationSchema {
      */
     @Config
     public static class SecondConfigurationSchema {
-        @Value
+        @Value(hasDefault = true)
         @Immutable
-        public int intCfg;
+        public int intCfg = 0;
 
-        @Value
-        public String strCfg;
+        @Value(hasDefault = true)
+        public String strCfg = "";
     }
```

Comment:
I think we can add fields with default values instead of adding default values for the current ones.

Reply:
Why are these changes needed in the first place?